packages/edit-site/src/components/editor/index.js

```diff
@@ -171,7 +165,7 @@ export default function Editor( { isLoading } ) {
 			// translators: A breadcrumb trail in browser tab. %1$s: title of template being edited, %2$s: type of template (Template or Template Part).
 			__( '%1$s ‹ %2$s ‹ Editor' ),
 			getTitle(),
-			typeLabels[ editedPostType ] ?? typeLabels.wp_template
+			POST_TYPE_LABELS[ editedPostType ] ?? POST_TYPE_LABELS.wp_template
```

Comment:
Nit: Might be better to do:

```suggestion
			POST_TYPE_LABELS[ editedPostType ] ?? POST_TYPE_LABELS[ TEMPLATE_POST_TYPE ]
```